docs/topics/db/managers.txt

```diff
@@ -454,3 +454,50 @@ However, if you're overriding ``__getattr__`` or some other private
 method of your ``Manager`` object that controls object state, you
 should ensure that you don't affect the ability of your ``Manager`` to
 be copied.
+
+Custom Managers with queries through relations
+----------------------------------------------
+
+Please consider that custom managers don't apply to intermediate joins.
+See following example::
+
+    class ItemManager(Manager):
+        use_for_related_fields = True
```

Comment:
@charettes thanks! changed and verified that behavior is corresponded to documentation